packages/react-scripts/package.json

```diff
@@ -10,7 +10,13 @@
   "bugs": {
     "url": "https://github.com/facebookincubator/create-react-app/issues"
   },
-  "files": ["bin", "config", "scripts", "template", "utils"],
+  "files": [
```

Comment:
Do you mind if we keep the formatting from CRA so we don't create future merge conflicts plz.